bach/bach/series/series_string.py

```diff
@@ -226,3 +228,66 @@ def __add__(self, other) -> 'Series':
 
     def _comparator_operation(self, other, comparator, other_dtypes=tuple(['string'])) -> 'SeriesBoolean':
         return super()._comparator_operation(other, comparator, other_dtypes)
+
+    def to_json_array(self, partition: WrappedPartition = None) -> 'SeriesJson':
+        """
+        Aggregate function: Concatenate the values of this Series into a json array
```

Comment:
IMHO `Concatenate` can get mixed up with "concatenating" arrays.
```suggestion
        Aggregate function: Group the values of this Series into a json array
```